scripts/fuzz.sh

```diff
@@ -39,7 +75,50 @@ main()
     -DCONFIG_ASAN=y "$@"
 
   mkdir -p ./fuzz_corpus
```

Comment:
Thanks for the review!

I can imagine a couple ways to preserve the corpus but they would take some work and I would like to merge this foundation first to avoid future, time-consuming regressions like #7318